zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1759,6 +1759,45 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /process-definitions/{processDefinitionId}/statistics/process-instances:
+    post:
+      tags:
+        - Process definition
+      operationId: getProcessDefinitionInstanceVersionStatistics
+      summary: Get statistics about process instances, grouped by process definition version
+      description: |
+        Get statistics about process instances, grouped by version for a given process definition.
+        For each version, the response includes the deployed instance name, version number,
+        total counts of active instances without incident, and total counts of active instances with incident.
```

Comment:
💭 I think we can leave this out of the description. We always have to maintain this, then. The response type defines what's returned. We can probably change this for the parent list as well, should we have it there.